include/swift/AST/Decl.h

```diff
@@ -780,6 +780,10 @@ class alignas(1 << DeclAlignInBits) Decl : public ASTAllocated<Decl> {
     /// Whether this module has been built with C++ interoperability enabled.
     HasCxxInteroperability : 1,
 
+    /// Whether this module uses the platform default C++ stdlib, or an
+    /// overridden C++ stdlib.
+    CXXStdlibKind : 2,
```

Comment:
This can lead to truncation of we add all enumerator. Should we just hold the space?

Reply:
Do you mean we should reserve more bits for this field?

Reply:
On the one hand, this is versioned, but on the other hand reserving (say) eight bits up front means we would likely never run into a problem that required us to update it. We would need to be a little careful about decoding unknown values for it, potentially, but I can't see a real downside.